doc/api/buffer.md

```diff
@@ -1211,6 +1211,8 @@ changes:
   - version: v5.7.0
     pr-url: https://github.com/nodejs/node/pull/4935
     description: The `encoding` parameter is supported now.
+  - version: REPLACEME
+    description: Throws `ERR_OUT_OF_RANGE` instead of `ERR_INDEX_OUT_OF_RANGE`.
```

Comment:
Missed `pr-url` field?

Reply:
Heh, yeah, I wasn't sure what the pr-url number would be, but I guess now I know. Will add it momentarily...

Reply:
I'll also move it to the top of the list because apparently these are in reverse chronological order...

Reply:
Added `pr-url` field.